### DISCUSSION TITLE
hcxdumptool

### DISCUSSION THREAD

**Discussion by blackai0** [ZerBea/hcxdumptool#383] [Q&A]
hcxdumptool: unrecognized option '--enable_status'
hcxdumptool 6.3.2-41-g23bf263 (C) 2023 by ZeroBeat
This is a penetration testing tool!
It is made to detect vulnerabilities in your NETWORK mercilessly!

plese give me solution

**Comment by hyekalhitech** [comment]


**Reply by hyekalhitech** [reply]
If you take look at this option list you'll see that there is no option "-o", no option "--active_beacon" and no option "--active_beacon --enable_status=15" any longer. This options are removed since hcxdumptool v6.3.0.

To get the same result on hcxdumptool >= v6.3.0 run this command:
$ hcxdumptool -i wlan0 -w dumpfile.pcapng -F --rds=1

**Comment by abakar2006** [comment]
![٢٠٢٤٠١٠٨_١٣٤٥٥٨](https://github.com/ZerBea/hcxdumptool/assets/155984271/bd762e38-ddd5-4bd0-b5bf-4bda17864403)


I've been stuck here for a week. I didn't want to cause a ridiculous problem in the professional community, but who else helps the self-maker

**Reply by ZerBea** [reply]
Option "o" has been replaced by option "w"
Option "active_beacon" has been replaced by option "disable_beacon"
Option "enable_status" has been replaced by option "rds"

Al this is mentioned in -h, changelog and [here](https://github.com/ZerBea/hcxdumptool/discussions/383#discussioncomment-7872230)

And additional you can find an example of a basic command line in --help.

**Reply by abakar2006** [reply]
Thank you very much. I found it. Sometimes the solution is self-evident, and yet the beginner searches in other places until he is guided
